src/queries/fake-fei-goals.sql

```diff
@@ -89,4 +89,8 @@ AND (NULLIF(current_setting('ssdi.status', true), '') IS NULL
           FROM json_array_elements_text(COALESCE(NULLIF(current_setting('ssdi.status', true), ''),'[]')::json) AS value
       ))
 GROUP BY 1,2,3,4,5,6,7,8,11,12
+-- Ghost goal filter
+HAVING NOT (g."createdVia" = 'activityReport'
+	AND COUNT(DISTINCT a.id) FILTER (WHERE a."calculatedStatus" = 'approved') = 0
```

Comment:
I didn't drill to much into the joins here, but why are we doing a count here instead of a simple != ?

Reply:
It has to aggregate so that it doesn't delete goals where some related ARs are approved and some aren't.